UsabillaSampleApp/README.md

```diff
@@ -26,4 +35,124 @@ $ cordova-res
 $ cordova-res ios
 ```
 
-Otherwise, if `config.xml` exists, `cordova-res` will look for platforms (e.g. `<platform name="ios">`) and generate resources only for the configured platforms.
\ No newline at end of file
+Otherwise, if `config.xml` exists, `cordova-res` will look for platforms (e.g. `<platform name="ios">`) and generate resources only for the configured platforms.
+
+### Add Platform
+
+- To add native platforms like android & ios , just run:
+
+```bash
+$ cordova platform add android ios
+```
+
+### Add Usabilla plugin
+
+- To add usabilla into cordova application , just run:
+
+```bash
+$ cordova plugin add usabilla-cordova --save
+```
+
+### Setup SDK
+
+-  Update Configurations to run sample app
+``` js
+<script type="text/javascript" src="js/config.js"></script>
+```
+``` js
+/// Usabilla Configuration
+const appId = 'YOUR_APP_ID_HERE';
+const formId = 'YOUR_FORM_ID_HERE';
+let event = 'YOUR_EVENT_TAG_HERE';
+const customVariable = {'YOUR_KEY_HERE': 'YOUR_VALUE_HERE'};
+```
+
+### Functions
+
+- Initialize the sdk and sets custom variables for targeting Campaigns.
+```
+initialize → String appId , Object customVariable
+```
+``` js
+    const customVariable = {'YOUR_KEY_HERE': 'YOUR_VALUE_HERE'};
+    Usabilla.initialize(function() {}, function (err) {}, appId, customVariable);
+```
+- Shows the Passive Form & Returns a JSON Object, contains result - rating, pageindex, sent flag.
```

Comment:
`Show the passive form and return a JSON Object containing the results - rating, pageIndex and sent flag`

Reply:
Done